### PR TITLE
feat: tx popup feedback

### PR DESCRIPTION
**Pull request by matt-user**
close FRO-643

### CODE REVIEW

File: packages/app/src/systems/Bridge/services/bridge.tsx [hunk 1]
```diff
@@ -95,6 +96,10 @@ export class BridgeService {
           store.openTxEthToFuel({
             txId,
           });
+          toast.success(
+            'Deposit successfully initiated.  You may now close the poup.',
```
Comment (LuizAsFight) [inline packages/app/src/systems/Bridge/services/bridge.tsx:100]:
```suggestion
            'Deposit successfully initiated. You may now close the popup.',
```

File: packages/app/src/systems/Bridge/services/bridge.tsx [hunk 2]
```diff
@@ -120,6 +125,10 @@ export class BridgeService {
           store.openTxFuelToEth({
             txId,
           });
+          toast.success(
+            'Withdraw successfully initiated.  You may now close the poup.',
```
Comment (LuizAsFight) [inline packages/app/src/systems/Bridge/services/bridge.tsx:129]:
```suggestion
            'Withdraw successfully initiated. You may now close the popup.',
```

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/Bridge/services/bridge.tsx [hunk 3]
```diff
@@ -95,6 +96,10 @@ export class BridgeService {
           store.openTxEthToFuel({
             txId,
           });
+          toast.success(
+            'Deposit successfully initiated. You may now close the popup.',
+            { duration: 5000 }
+          );
```
Comment (LuizAsFight) [inline packages/app/src/systems/Bridge/services/bridge.tsx:102]: I thought on this way could work, but actually if you check bridge service you will see at this point we don't know if the transaction is successful yet. we just grab txHash.

If the transaction is reverted or unsuccessful, the user will not know.

We should improve this approach:
- save in localStorage that this transaction ID was created from bridge form
- in the machine after we pass the first step of getting the `txResult (Fuel side)` or `txReceipts (ETH side)`, means we're sure the transaction worked. then show the success toast with this message


/\ this approach applies for both deposit/withdraw

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
wait for popup to open and make sure transaction is success

File: packages/app/src/systems/Bridge/services/bridge.tsx (hunk 3 above)
Comment (matt-user) [reply packages/app/src/systems/Bridge/services/bridge.tsx:102]: Waiting for the transaction to be successful isn't the point of this feature.  The point of the feature is to inform the user they can close the dialog without their transaction being canceled.  If we wait until the transaction is finished then there is no point to this feature.

Comment (LuizAsFight) [reply packages/app/src/systems/Bridge/services/bridge.tsx:102]: the transaction finished doesn't mean bridge transaction finished, just means the first step (tx on eth/fuel side finished).

if the user close the popup and the transaction is reverted, he will never know, and the tx will not show on bridge tx list

**Pull request merged**